action/menu.js

```diff
@@ -0,0 +1,36 @@
+const installedScripts = [
+  {name: 'reblog_timestamps', title: 'Reblog Timestamps'},
+  {name: 'all_features', title: 'All Features'},
+];
+
+const scriptsList = document.getElementById('scriptslist');
+const writeEnabled = () => {
+  const enabledScripts = [];
+
+  for (const listItem of scriptsList.children) {
+    const input = listItem.getElementsByTagName('input')[0];
+    if (input.checked === true) {
+      enabledScripts.push(input.id);
+    }
+  }
+
+  browser.storage.local.set(
+    {enabledScripts},
+  ).catch(console.error);
+};
+
+installedScripts.forEach(script => {
+  const listItem = document.createElement('li');
+  listItem.innerHTML = `
+    <input type="checkbox" id="${script.name}">
+    <label for="${script.name}">${script.title}</label>
+  `;
+
+  scriptsList.appendChild(listItem);
+  document.getElementById(script.name).oninput = writeEnabled;
```

Comment:
this would be cleaner if we made the input and label using createElement and then attached the event handlers directly, so we don't have to worry about ids and stuff.